cairo/tests/utils/evm_builder.py

```diff
@@ -92,6 +92,14 @@ def with_stack(self, strategy=stack_strategy(Stack[U256])):
         self._stack = strategy
         return self
 
+    def with_capped_values_stack(self, max_value=2**8 - 1):
```

Comment:
created this because otherwise 90% most of my tests were returning `ExceptionalHalt(OutOfBoundsRead)` and the rest with `StackUnderflow`